src/ctia/bundle/routes.clj

```diff
@@ -97,7 +98,8 @@
                  :query-params
                  [{external-key-prefixes
                    :- (describe s/Str "Comma separated list of external key prefixes")
-                   nil}]
+                   nil}
+                  {wait_for :- (describe s/Bool "wait for created entities to be available for search") nil}]
```

Comment:
I would not expose this query param for the bundle import route. It must be set to true due to the usage of a `pmap` in the `ctia.bulk.core` ns.